docs/stencil-docs/developing-further/google-analytics-enhanced-ecommerce.md

```diff
@@ -32,24 +32,23 @@ Cornerstone versions 2.6.0+ will have Data Tags already included in the theme.
 
 ## Prerequisites
 
-* BigCommerce store using your store’s checkout must be using Optimized One-Page Checkout.
+For this tutorial, you will need a BigCommerce store which uses Optimized One-Page Checkout for your store checkout.

```

Comment:
Recommendations:
* Put prereqs in the introduction and change to an h3
* Split into two prereqs
* Keep it concise. 

Example:

```
### Prerequisites
* [BigCommerce Store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial)
* [Optimized One-Page Checkout enabled](https://support.bigcommerce.com/s/article/Optimized-Single-Page-Checkout)
```